social-network/src/ForYou.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Box1 from './Box1';
+
+
+class ForYou extends React.Component {
+    render(){
+     return(   
+        <div>
+         <Box1 />
```

Comment:
Instead of making a separate component here that doesn't provide a lot of value, you may want to separate either the author or the entire article "box" into its own component. Note that those portions are repetitive and can be consolidated for reuse.

Reply:
I made them into separate components so I can style the divs with different CSS.  I see your point.  Instead of having Box1 for "ForYou.js" I should just make the two the same.  And refactor MissedIt.js and Box2 to be the same?

Reply:
looking at the example on the original assignment page.  So like you stated the authors bio since it's formatted the same on both ForYou and MissedIt that could be it's own component?

Reply:
You could do that, yes. Or just pull out the author if you'd rather have two separate articles w/different CSS.